standard/classes.md

```diff
@@ -2229,20 +2225,16 @@ In a method that takes reference parameters, it is possible for multiple names t
 >
 > *end example*
 
-#### 15.6.2.5 Output parameters
-
-A parameter declared with an `out` modifier is an output parameter. An output parameter is a local reference variable ([§9.7](variables.md#97-reference-variables-and-returns)) that gets its initial referent from the corresponding argument supplied in the method invocation.
-
-When a formal parameter is an output parameter, the corresponding argument in a method invocation shall consist of the keyword `out` followed by a *variable_reference* ([§9.5](variables.md#95-variable-references)) of the same type as the formal parameter. A variable need not be definitely assigned before it can be passed as an output parameter, but following an invocation where a variable was passed as an output parameter, the variable is considered definitely assigned.
+##### 15.6.2.5 Output parameters
 
-Within a method, just like a local variable, an output parameter is initially considered unassigned and shall be definitely assigned before its value is used.
+A parameter declared with an `out` modifier is an output parameter. Within a method, just like a local variable, an output parameter is initially considered unassigned and shall be definitely assigned before its value is used. Every output parameter of a method shall be definitely assigned before the method returns.
 
-Every output parameter of a method shall be definitely assigned before the method returns.
+A method declared as a partial method ([§15.6.9](classes.md#1569-partial-methods)) may not have output parameters.
 
-A method declared as a partial method ([§15.6.9](classes.md#1569-partial-methods)) or an iterator ([§15.14](classes.md#1514-iterators)) may not have output parameters.
-
-Output parameters are typically used in methods that produce multiple return values.
+> *Note: Output parameters are typically used in methods that produce multiple return values. *end note*
```

Comment:
This was normative in the existing draft. I don't think it's required to be normative, so I made this a note.

Reply:
Agreed, anything with "typically" shouldn't be normative.